corehq/ex-submodules/phonelog/migrations/0008_devicelog_varchar_index.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-06-01 18:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunSQL
+from corehq.util.django_migrations import add_if_not_exists
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0007_devicelog_indexes'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            add_if_not_exists(
```

Comment:
I think this is the wrong way to use this migration. I think it should be like https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/phonelog/migrations/0007_devicelog_indexes.py

It was used incorrectly in accounting and I think it cause a headache on deploy or locally although I don't remember exactly what